OptimizelySDK/Optimizely.cs

```diff
@@ -262,6 +265,8 @@ public void Track(string eventKey, string userId, UserAttributes userAttributes
                     Logger.Log(LogLevel.ERROR, string.Format("Unable to dispatch conversion event. Error {0}", exception.Message));
                 }
 
+                NotificationCenter.FireNotifications(NotificationCenter.NotificationType.Track, eventKey, userId, 
```

Comment:
@mfahadahmed  please check firenotification should be called on success of dispatchevent or anyhow it should be called.
@oakbani FYI.

Reply:
FireNotifications should be called whenever we attempt to dispatch event. Success or failure should not be considered for firing notifications.